Dockerfile.custom

```
@@ -45,30 +45,18 @@
 
 # Pin kubectl minor version (must be within 1 minor version of cluster version)
 # Note, however, that due to Docker layer caching and the structure of this
 # particular Dockerfile, the patch version will not automatically update
 # until you change the minor version or change the base Geodesic version.
 # If you want, you can pin the patch level so you can update it when desired.
-ARG KUBECTL_VERSION=1.20
+ARG KUBECTL_VERSION=1.30
 RUN apt-get update && apt-get install kubectl-${KUBECTL_VERSION}
 
 # Install Atmos CLI (https://github.com/cloudposse/atmos)
 RUN apt-get install atmos
```

Hadolint (2.12.0)

[info] Avoid additional packages by specifying `--no-install-recommends`

(DL3015)